rpmvenv/extensions/python/venv.py

```diff
@@ -121,21 +121,22 @@ def generate(config, spec):
 
         spec.blocks.files.append('/%{venv_install_dir}')
 
-        spec.blocks.install.append('%{venv_cmd} %{venv_dir}')
+        spec.blocks.install.extend((
+            '%{venv_cmd} %{venv_dir}',
+            'cd %{SOURCE0}',
+        ))
         for requirement in config.python_venv.requirements:
 
             spec.blocks.install.append(
-                '%{{venv_pip}} -r %{{SOURCE0}}/{0}'.format(
+                '%{{venv_pip}} -r {0}'.format(
                     requirement,
                 )
             )
 
         if config.python_venv.require_setup_py:
-            spec.blocks.install.extend((
-                'cd %{SOURCE0}',
-                '%{venv_python} setup.py install',
-                'cd -',
```

Comment:
I'm fairly sure that most plugins use the macros provided which all contain absolute paths, but removing this `cd -` line does change the working directory for future lines executing in the `install` block. If someone has a private and/or 3rd party plugin then this might break them.

Would you be open to either surrounding each requirement and setup.py with `cd %{SOURCE0}` and `cd -` or a `cd -` right before the line 140/141? It seems small, but I'm trying to be extra cautious with this to make sure we don't break any builds.